jdi-light-material-ui-tests/src/main/java/io/github/com/StaticSite.java

```diff
@@ -456,16 +456,7 @@
 
     // popover
     @Frame("storybook-preview-iframe")
-    public static PopoverClickFrame popoverClickFrame;
-
-    @Url("/material-ui-utils-popover--click")
-    public static WebPage utilsPopoverClickPage;
-
-    @Frame("storybook-preview-iframe")
-    public static PopoverHoverFrame popoverHoverFrame;
-
-    @Url("/material-ui-utils-popover--hover")
-    public static WebPage utilsPopoverHoverPage;
+    public static PopoverPage popoverClickFrame;
```

Comment:
Please use popoverPage instead of popoverClickFrame (issues with "Click" and "Frame" words)